apps/site/react_renderer/app.js

```diff
@@ -11,6 +11,20 @@ import ScheduleDirection from "../assets/ts/schedule/components/ScheduleDirectio
 import TripPlannerResults from "../assets/ts/trip-plan-results/components/TripPlannerResults";
 import MoreProjectsTable from "../assets/ts/projects/components/MoreProjectsTable";
 
+// log the process id when a process is started
+process.stdout.write(`node_logging node_process process=${process.pid}\n`);
```

Comment:
@meagonqz we're logging just enough messages, and their format is regular enough, that I'd like to see a helper that we could invoke here like `log("node_process", {process: process.pid})`, which would do all the formatting for us.